data/tribes/scripting/help/building_help.lua

```diff
@@ -889,41 +730,88 @@ end
 function building_help_production_section(tribe, building_description)
    -- Produced items
    local result = h2(_("Production"))
-   if (building_description.output_ware_types[1] or building_description.output_worker_types[1]) then
-      local checked_programs ={}
-      for i, ware_description in ipairs(building_description.output_ware_types) do
-         programs, ware_counters, ware_strings = programs_wares_count(tribe, building_description, ware_description)
-         -- check if the ware is collected (no producing program)
-         if #programs == 0 then
-            result = result .. h3(_("Ware produced:"))
-            result = result .. help_ware_amount_line(ware_description, 1)
-            break
-         end
+   local checked_programs = {}
+   if (building_description.type_name == "trainingsite") then
+      programs, ware_counters, ware_strings = programs_wares_count(tribe, building_description, nil)
+      if (not checked_programs[program]) then
          for j, program in ipairs(programs) do
-            if (ware_counters[program] > 0) and not checked_programs[program] then
+            trained = building_description:trained_soldiers(program)
+            -- TRANSLATORS: Building Encyclopedia Skill trained by a trainingsite
+            if (trained[1] == "Health") then
```

Comment:
This is not a translatable string

Reply:
correct. It is just a string value written from c++ to lua which is not intended to be used in a translatable string. I could have used an arbitrary enum but I thought a string would be more human readable in the code.